<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminpage</class>
 <widget class="QDialog" name="Adminpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnDisplay">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Display Lost Items</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FINDER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemove">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Remove Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DESCRIPTION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;
color:white;
background-color:rgb(64, 1, 11); 
font:&quot;Arial&quot;; 
border-radius:20px;}
QPushButton:Hover
{color:black;
background-color:rgb(242, 201, 96);
border-radius:50px;}
QPushButton:pressed
{background-color:green;}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="lineEditFinder">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>180</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textEditDescription">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>481</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>681</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 18, 92);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>btnDisplay</zorder>
  <zorder>label_2</zorder>
  <zorder>btnRemove</zorder>
  <zorder>label_3</zorder>
  <zorder>btnExit</zorder>
  <zorder>label</zorder>
  <zorder>lineEditFinder</zorder>
  <zorder>textEditDescription</zorder>
  <zorder>lineEditName</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
